runelite-client/src/main/java/net/runelite/client/plugins/deathindicator/DeathWorldMapPoint.java

```diff
@@ -37,11 +40,25 @@
 
 	static
 	{
-		WORLDMAP_HINT_ARROW = new BufferedImage(DeathIndicatorPlugin.MAP_ARROW.getWidth(), DeathIndicatorPlugin.MAP_ARROW.getHeight(), BufferedImage.TYPE_INT_ARGB);
+		BufferedImage MAP_ARROW;
+		try
+		{
+			synchronized (ImageIO.class)
+			{
+				MAP_ARROW = ImageIO.read(DeathWorldMapPoint.class.getResourceAsStream("clue_arrow.png"));
```

Comment:
I think this image is non-existant (you added clue_scroll instead of clue_arrow). Also, probably as now same image is used at 2 places, and probably will be at more in future, maybe moving the clue_arrow to some top level  package and naming it like map_arrow and reusing in both this and clue scroll plugin might make sense.